ykman-gui/qml/PivSetManagementKeyView.qml

```diff
@@ -205,14 +182,47 @@ ColumnLayout {
                     ToolTip.delay: 1000
                     ToolTip.visible: hovered
                     ToolTip.text: qsTr("Management key must be exactly 48 hexadecimal digits.")
-                    visible: hasNewManagementKeyInput
                 }
                 CustomButton {
                     id: randomManagementKeyBtn
                     text: qsTr("Generate")
                     Layout.alignment: Qt.AlignHCenter | Qt.AlignVCenter
                     onClicked: generateManagementKey()
-                    visible: hasNewManagementKeyInput
+                }
+
+                Label {}
```

Comment:
Why the empty Labels?

Reply:
To make the GridLayout line things up correctly...